workflows/diagnostics/fv3net/diagnostics/prognostic_run/load_run_data.py

```diff
@@ -105,6 +106,14 @@ def load_grid(catalog):
     return xr.merge([grid_c48, ls_mask])
 
 
+def _drop_grid_if_present(
+    ds: xr.Dataset,
+    grid_vars: Sequence[str] = constants.GRID_VARS,
+    grid_interface_coords: Sequence[str] = constants.GRID_INTERFACE_COORDS,
+) -> xr.Dataset:
+    return ds.drop_vars(grid_vars + grid_interface_coords, errors="ignore")
```

Comment:
It's a one liner so could just copy paste it below IMO and avoid making a function with several args. Up to you.